### ISSUE TITLE
Question about domains

### ISSUE THREAD

**Issue by jcelerier** [OSSIA/libossia#37]
I don't quite understand the code in  `Address & JamomaAddress::setDomain(shared_ptr<Domain> domain)` (Address.cpp:235)

If I give a domain with  : 

```
min = 0
max = 10 
range = [5, 15]
```

doesn't it cause the actual min and max to be 5 and 15 ? (while I would expect to just be able to send the value "5") shouldn't them be prepended or something ?

**Comment by bltzr** [comment]
I guess that's a question for @theod right ?

**Comment by theod** [comment]
AFAIR the range was to restrict the Domain to some precise value only.
user example : if you have a parameter dedicated to set the bauds rate, you have to choose between standard values (..., 9600, 14400, 19200, ..., 57600, 115200, ...).

but maybe the name of the attribute is not good considering this purpose ?

**Comment by jcelerier** [comment]
yes, I renamed it into "values". What wasn't clear for me was : 

* In minuit, how to give a range with only two possible values, without giving a min and max ? For instance 44100 or 48000 ? Because from what I understood of the code at the time, it always takes the first value for min, the second for max, and the remaining for the "allowed" values.

* Do the min / max also apply to the "allowed" values ?

by the way Theo, I changed a lot the API this summer (rewrote a big part of it without Jamoma, and without inheritance mostly for performance reasons), if you want we can do a skype at some point so that I tell you what changed.
